drivers/hmis/app/graphql/types/application/activity_log.rb

```diff
@@ -0,0 +1,35 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class Application::ActivityLog < Types::BaseObject
+    # maps to Hmis::ActivityLog
+    graphql_name 'ActivityLog'
+    field :id, ID, null: false
+    field :user, Types::Application::User, null: false
+    field :ip_address, String, null: true
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :resolved_records, [Types::Application::ActivityLogRecord], null: false
+
+    def user
+      load_ar_association(object, :user)
+    end
+
+    def resolved_records
+      object.resolved_fields.keys.map do |key|
+        record_type, record_id = key.split('/')
```

Comment:
this is fine as as. You can also provide a second arg to split just in-case there's an extra '/' in key somehow
```suggestion
        record_type, record_id = key.split('/', 2)
```